tools/mksurfdata_esmf/gen_mksurfdata_build.sh

```diff
@@ -7,10 +7,17 @@ case $hostname in
   ##cheyenne
   cheyenne* | r* )
       export MACH="cheyenne"
+      pio_iotype=1
+      ;;
+  ##casper
+  casper* )
+      export MACH="casper"
+      pio_iotype=1
       ;;
   ## Other machines
   *)
       export MACH="$hostname"
+      pio_iotype=2
```

Comment:
@ekluzek recommendation:
Make izumi and hobart explicit (under separate checks).
Keep "other machines" as is and add note that says "we assume that pnetcdf is off; therefore, pio_iotype = 2."

Reply:
clm_pymod testing IN PROGRESS
python testing OK
2 lilac tests fail because I'm working with ...dev090

Also confirm that mksurfdata_esmf still works on casper & izumi before merging PR to mksurfdata branch.